[2025-09-13T13:25:49.826+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-13T13:25:49.864+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-12T00:00:00+00:00 [queued]>
[2025-09-13T13:25:49.882+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-12T00:00:00+00:00 [queued]>
[2025-09-13T13:25:49.884+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-13T13:25:49.914+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task_3> on 2025-09-12 00:00:00+00:00
[2025-09-13T13:25:49.927+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_verify', 'task_3', 'scheduled__2025-09-12T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/dag_verify.py', '--cfg-path', '/tmp/tmp9ya4pxqp']
[2025-09-13T13:25:49.936+0000] {standard_task_runner.py:105} INFO - Job 127: Subtask task_3
[2025-09-13T13:25:49.940+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=206) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-13T13:25:49.941+0000] {standard_task_runner.py:72} INFO - Started process 222 to run task
[2025-09-13T13:25:50.045+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_verify.task_3 scheduled__2025-09-12T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-13T13:25:50.240+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_verify' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-12T00:00:00+00:00'
[2025-09-13T13:25:50.241+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-13T13:25:50.776+0000] {logging_mixin.py:190} INFO - Matplotlib version: 3.9.2
[2025-09-13T13:25:50.777+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-13T13:25:50.795+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-13T13:25:50.796+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_verify, task_id=task_3, run_id=scheduled__2025-09-12T00:00:00+00:00, execution_date=20250912T000000, start_date=20250913T132549, end_date=20250913T132550
[2025-09-13T13:25:50.883+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-13T13:25:50.912+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-13T13:25:50.916+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
